tests/user_tests/test_user_api.py

```diff
@@ -0,0 +1,22 @@
+from tests.test_base import TestBase
+from app.start import db
+from app.models import User
+import json
+
+class TestUserAPI(TestBase.TestFlaskWithDB):
+    def setUp(self):
+        super().setUp()
+        
+        self.user = User(name='Test User', email='test@user.com')
```

Comment:
This should be its own test. You can order the tests to ensure that it gets called first.

Reply:
Test should generally be independent of execution order and this uses a temporary transaction that is rollback'd at the end of the test anyway.

Reply:
Yes, independent tests are nice. What I'm saying is, adding a user should be its own test, as well as setup for querying a user.